### PR TITLE
OIDC page alignment fix

### PR DESCRIPTION
**Pull request by nsuma8989**
This PR is trying to fix the alignment errors with the OIDC page

### CODE REVIEW

**Comment by nsuma8989** [comment]
@mbelton-buildkite We got user complain about the alignment issues with the page so created this PR. Please can you check on this so we can update it

File: pages/agent/v3/help/_oidc_claims.md
Comment (mbelton-buildkite) [inline pages/agent/v3/help/_oidc_claims.md:1]: What's the reason for moving this to a separate file? The other files in this directory are all generated from the CLI while this is not

**Review by mbelton-buildkite — COMMENTED** [review]
The change to the page title and first paragraph look all good, but just want to clarify the reason for moving the content to separate file.

File: pages/agent/v3/help/_oidc_claims.md
Comment (nsuma8989) [reply pages/agent/v3/help/_oidc_claims.md:1]: No, I thought we keep all the table contents into separate file and was not aware that files in that other directory are generated from CLI. I will move it back to main page.

Comment (nsuma8989) [reply pages/agent/v3/help/_oidc_claims.md:1]: Updated the changes as suggested. Please can you review the claims section to see if alignment of the page looks fine.

File: pages/agent/v3/cli_oidc.md [hunk 1]
```diff
@@ -170,6 +183,7 @@ Generate these additional claims by adding `--claims` to the `buildkite-agent oi
   </tbody>
 </table>
 
+
```
Comment (mbelton-buildkite) [inline pages/agent/v3/cli_oidc.md:186]: I think this blank line is a typo?

```suggestion
```

**Review by mbelton-buildkite — APPROVED** [review]
Looks good! Thanks for fixing this :)

File: pages/agent/v3/cli_oidc.md (hunk 1 above)
Comment (nsuma8989) [reply pages/agent/v3/cli_oidc.md]: Yes, cleaned it up

**Pull request merged**